docs/db-cloud-shared/tutorials/monitor-host-metrics/go-demo.md

```diff
@@ -0,0 +1,59 @@
+
+In this section, we will create a quick start demo and showcase the core code to collect host metrics and send them to Greptime. The demo is based on [OTLP/HTTP](https://opentelemetry.io/). For reference, you can obtain the entire demo on [Github](https://github.com/GreptimeCloudStarters/quick-start-go).
+
+To begin, create a new directory named `quick-start-go` to host our project. Then, run the command `go mod init quick-start` in the directory from your terminal. This will create a `go.mod` file, which is used by Go to manage imports.
```

Comment:
```suggestion
To begin, create a new directory named `quick-start-go` to host our project. Then, run the command `go mod init quick-start` in the directory from your terminal. This will generate a `go.mod` file, which is used by Go to manage imports.
```